lib/windows/app/components/SerialPortSelector.jsx

```diff
@@ -175,6 +185,7 @@ SerialPortSelector.propTypes = {
     dropdownCssClass: PropTypes.string,
     dropdownMenuCssClass: PropTypes.string,
     menuItemCssClass: PropTypes.string,
+    filterPort: PropTypes.func.isRequired,
```

Comment:
I was wrong. The function only receives a single port - not an array. As discussed with Bence we could consider renaming to just `filter` though.